### PR TITLE
support for setting custom User Agent added

### PR DESCRIPTION
**Pull request by rozie**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
Added support for setting custom User Agent for whole integration (defined by API key) and for single requests.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by ShacharKidor** [comment]
Hi @rozie , we appreciate your effort on contributing to the XSOAR marketplace. Due to excessive amount of contributions in our queue, it may take us a bit more time than usual to review this pull request, we appreciate your patience.

**Review by ShacharKidor — CHANGES_REQUESTED** [review]
Good Job!
@rozie - Thank you for contributing. 

In order to complete your contribution, It would be great if you'll update the README.md file of the integration with the new argument. 
The sections that should be updated are: 
* Configuration section at the beginning of the readme file. 
* The following commands: 'urlscan-submit' and 'url'.

Apologies again for the delay in review, please let me know if you need any help.

**Comment by rozie** [comment]
Hi,
no problem with delay. :)
README.md updated. Let me know if this is enough.

**Comment by ShacharKidor** [comment]
> Hi, no problem with delay. :) README.md updated. Let me know if this is enough.

@rozie - Hi thank you :) saw your updates.
You missed the 'Configure urlscan.io on Cortex XSOAR section' of the readme file - Please add the new parameter to there as well.

**Comment by rozie** [comment]
Done. :)

**Comment by ShacharKidor** [comment]
> Done. :)

@rozie - great!
Are you available for a quick demo before merging PR? Let's schedule

In addition - Please add Release notes describing the change you added, by using the demisto-sdk update-release-notes command.
For more information see [release-notes](https://xsoar.pan.dev/docs/documentation/release-notesl)

Please let me know if you need any help :)

**Comment by ShacharKidor** [comment]
@rozie - You can also grant me permissions to your branch and I'll add the Release notes for you. 
Please see [inviting-collaborators-to-a-personal-repository](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-u[…]-repositories/inviting-collaborators-to-a-personal-repository)

**Comment by rozie** [comment]
@ShacharKidor Trying to reach you at Slack. I'm having a little problem with adding Release notes myself and I'd like to solve it for the future.

**Comment by ShacharKidor** [comment]
> @ShacharKidor Trying to reach you at Slack. I'm having a little problem with adding Release notes myself and I'd like to solve it for the future.

I'll be happy to help - Let's schedule a zoom meeting.

File: Packs/UrlScan/Integrations/UrlScan/UrlScan.yml [hunk 1]
```diff
@@ -142,6 +147,8 @@ script:
       defaultValue: 'false'
       required: false
       secret: false
+    - name: useragent
+      description: User agent to perform request
```
Comment (ShacharKidor) [inline Packs/UrlScan/Integrations/UrlScan/UrlScan.yml:151]:
```suggestion
    - default: false
      isArray: false
      name: useragent
      description: User agent to perform request
      required: false
      secret: false
```
I added it just to keep the structure of the existing arguments

File: Packs/UrlScan/Integrations/UrlScan/UrlScan.yml [hunk 2]
```diff
@@ -271,6 +278,8 @@ script:
...
```
Comment (ShacharKidor) [inline Packs/UrlScan/Integrations/UrlScan/UrlScan.yml:282]:
```suggestion
    - default: false
      isArray: false
      name: useragent
      description: User agent to perform request
      required: false
      secret: false
```
I added it just to keep the structure of the existing arguments

File: Packs/UrlScan/ReleaseNotes/1_1_13.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### urlscan.io
+- Support for setting custom User Agent added
```
Comment (ShacharKidor) [inline Packs/UrlScan/ReleaseNotes/1_1_13.md:4]:
```suggestion
- Added a **useragent** parameter which enables setting a custom user agent used during scans with the integration.
- Added a **useragent** argument to ***urlscan-submit*** and ***url*** commands, which enables setting a custom user agent for scans which are performed using those commands. 
```

**Review by ShacharKidor — CHANGES_REQUESTED** [review]

**Review by ShacharKidor — APPROVED** [review]

**Pull request merged**